src/app/components/todolist/todolist.component.ts

```diff
@@ -22,7 +22,13 @@ export class TodolistComponent implements OnInit {
     // TODO return the array of todos from local storage
     this.todoTitle = '';
     this.idForTodo = 0;
-    this.todos = []
+    this.todos = this.getTodos();
+  }
+
+  getTodos(): Todo[] {
+    var retrievedTodos = JSON.parse(localStorage.getItem("todos"));
```

Comment:
Instead of accessing `localStorage` directly, can we please wrap it in a service, if we were to switch out the underling storage technology e.g. DB it would be easier to change the single service rather than everywhere we are using `localStorage`

Reply:
I've created a localstoragemanager service which can write to and get from the local storage.